Cargo.toml

```diff
@@ -31,5 +31,5 @@ tonic = { version = "0.10", features = ["transport", "tls", "tls-roots"] }
 
 [workspace.dependencies.deltalake]
 git = "https://github.com/delta-io/delta-rs.git"
-branch = "main"
+rev = "0ad02d4243d1e207f8e54c7b763a7d84f8143573"
```

Comment:
https://github.com/delta-io/delta-rs/commit/0ad02d4243d1e207f8e54c7b763a7d84f8143573

This seems old. Do we want to bump?

Reply:
yeah i just grabbed the one out of the cargo.lock. Adding lance messed up my dependencies. Ill see if i can bump it to a more recent revision.